app/models/user.rb

```diff
@@ -468,6 +468,20 @@ def seed_aspects
     aq
   end
 
+  def send_welcome_message
+    unless AppConfig.settings.welcome_message? && AppConfig.admins.account? then return end
```

Comment:
The syntax here is a bit confusing imho, what about more verbosity?

    unless AppConfig.settings.welcome_message? && AppConfig.admins.account?
      return
    end